docs/resources/instancev2.md

```diff
@@ -443,29 +474,35 @@ Optional:
 - `type` (String) Available value is 'subnet', 'any_subnet', 'external', 'reserved_fixed_ip'
 
 
-<a id="nestedblock--configuration"></a>
-### Nested Schema for `configuration`
+<a id="nestedblock--volume"></a>
+### Nested Schema for `volume`
 
 Required:
 
-- `key` (String)
-- `value` (String)
-
-
-<a id="nestedblock--volume"></a>
-### Nested Schema for `volume`
+- `volume_id` (String)
 
 Optional:
 
-- `attachment_tag` (String)
 - `boot_index` (Number) If boot_index==0 volumes can not detached
-- `delete_on_termination` (Boolean)
+
+Read-Only:
+
+- `attachment_tag` (String) Tag for the volume attachment
```

Comment:
I guess you can send this one on volume creation

Reply:
made it computed